AVL tree.cpp

```diff
@@ -23,9 +23,9 @@ class AVLtree {
 public:
     AVLtree(void);
     ~AVLtree(void);
-    bool insert(T key);
+    void insert(T key);
```

Comment:
i suggest using void type instead of bool

Reply:
Got it, I will make the changes tonight! Thanks.